progress_store.go

```diff
@@ -31,17 +30,17 @@ func (ss *SqlStore) setup(ctx context.Context, driver, datasource string) (func(
 	return db.Close, nil
 }
 
-func (ss *SqlStore) save(ctx context.Context, pipeline, msg_id string, progress int, publishTime time.Time, f func() error) error {
+func (ss *SqlStore) save(ctx context.Context, pipeline string, msg *Message, f func() error) error {
 	err := ss.transaction(func(tx *sql.Tx) error {
-		_, err := tx.Exec(SQL_UPDATE_JOBS, progress, msg_id, progress)
+		_, err := tx.Exec(SQL_UPDATE_JOBS, msg.progress, msg.msg_id, msg.progress)
 		if err != nil {
-			fmt.Println("Failed to update pipeline_jobs message_id: %v to status: %v cause of %v", msg_id, progress, err)
+			fmt.Println("Failed to update pipeline_jobs job_message_id: %v to progress: %v cause of %v", msg.msg_id, msg.progress, err)
```

Comment:
@minimum2scp Yes. I added `make check` task to find problems like this. And I fixed it.